Makefile

```diff
@@ -29,9 +38,23 @@ req:
 install: prepare
 	$(PREFIX)pip install .
 
+# https://developer.atlassian.com/docs/getting-started/set-up-the-atlassian-plugin-sdk-and-build-a-project/install-the-atlassian-sdk-on-a-linux-or-mac-system#InstalltheAtlassianSDKonaLinuxorMacsystem-Homebrew
 install-sdk:
-	# https://developer.atlassian.com/docs/getting-started/set-up-the-atlassian-plugin-sdk-and-build-a-project/install-the-atlassian-sdk-on-a-linux-or-mac-system#InstalltheAtlassianSDKonaLinuxorMacsystem-Homebrew
+ifeq ($(PLATFORM), darwin)
 	which atlas-run-standalone || brew tap atlassian/tap && brew install atlassian/tap/atlassian-plugin-sdk
+else ifneq ($(wildcard /etc/debian_version),)
+	sudo sh -c 'echo "deb https://sdkrepo.atlassian.com/debian/ stable contrib" >/etc/apt/sources.list.d/atlassian_development.list'
+	sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B07804338C015B73
+	sudo apt-get install apt-transport-https
+	sudo apt-get update
+	sudo apt-get install atlassian-plugin-sdk
+else ifneq ($(wildcard /etc/redhat-release),)
+	curl https://marketplace.atlassian.com/download/plugins/atlassian-plugin-sdk-rpm/version/42380 -o /tmp/atlassian-plugin-sdk.noarch.rpm
```

Comment:
The specification destination location is not secure. Please create a temporary directory to save it under using `mktemp -d`

Reply:
Thanks for the comments!!  Yes, will do this  . . .